pkg/test/framework/components/echo/echotest/run.go

```diff
@@ -19,16 +19,19 @@ import (
 
 	"istio.io/istio/pilot/pkg/networking/util"
 	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/cluster"
 	"istio.io/istio/pkg/test/framework/components/echo"
 )
 
 type (
 	perDeploymentTest  func(t framework.TestContext, instances echo.Instances)
 	perNDeploymentTest func(t framework.TestContext, deployments echo.Services)
 	perInstanceTest    func(t framework.TestContext, inst echo.Instance)
+	perClusterTest     func(t framework.TestContext, c cluster.Cluster)
 
-	oneToOneTest func(t framework.TestContext, src echo.Instance, dst echo.Instances)
-	oneToNTest   func(t framework.TestContext, src echo.Instance, dsts echo.Services)
+	oneToOneTest      func(t framework.TestContext, src echo.Instance, dst echo.Instances)
+	oneToNTest        func(t framework.TestContext, src echo.Instance, dsts echo.Services)
+	oneClusterOneTest func(t framework.TestContext, src cluster.Cluster, dst echo.Instances)
```

Comment:
I don't understand the `OneTest` part of the name. Isn't it `perClusterTest`?

Reply:
`perClusterTest` is already defined above at line 30.
The flow is
```
RunFromClusters(testFn oneClusterOneTest) {
  t.fromEachCluster(ctx, perClusterTest(){
    t.toEachDeployment(ctx, perDeploymentTest(){
    })
  })
}
```